### ISSUE TITLE
Autoinflammatory disease 

### ISSUE THREAD

**Issue by johannaest** [obophenotype/human-phenotype-ontology#6070]
**Preferred term label:**
Autoinflammatory disease 

**Synonyms**
AIDs

**Definition (free text, please give PubMed ID)**
Autoinflammatory diseases are a group of clinical conditions other than autoimmune diseases, characterized by recurrent inflammatory episodes. They are determined by a dysregulation of innate immunity, without involvement of specific immunity.
PMID: 24164192

**Parent term (use hpo.jax.org/app)**
 Abnormal inflammatory response

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 617099

**Gene association**
OTULIN (OMIM: 615712)

**Comment by pnrobinson** [comment]
Autoinflammation is not a phenotypic observation, it is an inference about the cause of a disease -- therefore, this is not in scope for the HPO. The HPO would try to annotate to the things that are directly observed.
